syntax = "proto3";

//import "pdcp.proto";
package stored;
message stored{
    message SNBuffer{
        repeated int32 sn = 1;
        int32 bufferSize = 2;
    }
    SNBuffer buffer = 1;
}
